config/environment.js

```diff
@@ -1,5 +1,9 @@
 'use strict';
 
 module.exports = function(/* environment, appConfig */) {
-  return { };
+  return {
+    emberResponsiveDefaults: {
+      autoInject: true
```

Comment:
That's how I did it initially, but the auto-injection of `view` and injecting it into the `media` property were things I wanted to get away from, a generated initializer would help new users and give advanced users an easy place to modify injections.